dev/drivers/scripts/stats/glwu/jevs_glwu_wave_grid2obs_stat.sh

```diff
@@ -0,0 +1,73 @@
+#PBS -N jevs_glwu_wave_grid2obs_stats
+#PBS -j oe
+#PBS -S /bin/bash
```

Comment:
@SamiraArdani-NOAA Again, please be sure to remove glwu files from this PR.  That should be throughout this PR.